### ISSUE TITLE
Include Strava API responses (ExVCR cassettes) for unit tests to support CI builds

### ISSUE THREAD

**Issue by boone** [slashdotdash/strava#1]
Any thoughts about including the ExVCR cassettes results in the repo, so you could add CI testing?

BTW, when running these tests, I found a bug in ExVCR where the [header blacklist](https://github.com/slashdotdash/strava/blob/master/config/test.exs#L8) wasn't working. I submitted a [fix](https://github.com/parroty/exvcr/pull/59) over there and was been released as 0.7.4.

**Comment by slashdotdash** [comment]
@boone Thanks for fixing that ExVCR bug. I'd noticed that the headers weren't being removed but didn't investigate any further. I chose to exclude the cassettes from git due to not wanting to accidentally commit the cookie header. CI is a good reason to include the files.

**Comment by boone** [comment]
I looked into this a bit. The updated ExVCR looks to be working as expected. But it's not clear to me which Strava API data is suitable for committing as test data. It seems like it wouldn't be a good idea to just commit any data with random user information.

Strava's own Go client has [cassette data](https://github.com/strava/go.strava/tree/master/cassettes), but it's a different format. I thought it would be a matter of picking the test IDs out of that data. But when I tried that on the club member list, the data I got was slightly different than the Go cassette, so I'm not sure if their cassettes were actually from API results and the data has changed, or they have edited them for the test suite.
